### ISSUE TITLE
docker/libcompose#319 is closed, update to `README.md`?

### ISSUE THREAD

**Issue by sanmai-NL** [dansteen/controlled-compose#4]
In the current `README.md` you state you're waiting on docker/libcompose#319. That last issue is closed, are you about update to `README.md`?

**Comment by dansteen** [comment]
Yep.   I'm going to circle back around to this as soon as I'm done with this tool: https://github.com/dansteen/constellation  (it's controlled-compose for rkt)  Should be done pretty soon.  I'm working on documentation today.
